geniza/footnotes/admin.py

```diff
@@ -39,7 +87,7 @@ class SourceAdmin(TabbedTranslationAdmin, admin.ModelAdmin):
     fields = ("source_type", "title", "year", "edition", "volume", "languages", "notes")
```

Comment:
(not related to this PR, surprised this got past QA) The SourceAdmin `list_display` has no clickable item where a user can then see that source's changeview. At least it's possible to construct a Source with fields that offer no way to select it from the list view

Reply:
oh — you mean it's possible to create a source without an author, which would give you nothing to click?

I think current data should always have an author (although we may have a handful where we have lost the information and can't track it down), but I do remember runing into this. Nrmally django should provide a default like "-" — I wondered if it wasn't doing this because author is a model translation field...